specification/netapp/resource-manager/readme.go.md

```diff
@@ -33,6 +34,15 @@ batch:
   - tag: package-2017-08-15
 ```
 
+### Tag: package-netapp-2021-04-01 and go
+
+These settings apply only when `--tag=package-netapp-2021-04-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-netapp-2021-04-01' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-04-01/$(namespace)
```

Comment:
We need to remove the `preview` here since this will  be a stable package
```suggestion
output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2021-04-01/$(namespace)
```